src/Game/Game.cpp

```diff
@@ -448,5 +446,19 @@ Audio::Mixer* Game::mixer()
     return _mixer.get();
 }
 
+Event::Dispatcher& Game::eventDispatcher()
+{
+    return _eventDispatcher;
+}
+
+}
+
+namespace Event
+{
+// static
+Dispatcher* Dispatcher::getInstance()
+{
+    return &Game::getInstance()->eventDispatcher();
+}
```

Comment:
That's a pseudo-singleton implementation.
I think that knowledge about Game inside Dispatcher.cpp is overkill.